packages/react/test/specs/lib/felaRenderer-test.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import { createSnapshot } from 'jest-react-fela'
+import { EmptyThemeProvider } from 'test/utils'
+import Box from 'src/components/Box/Box'
+import Animation from 'src/components/Animation/Animation'
+import Icon from 'src/components/Icon/Icon'
+import Provider from 'src/components/Provider/Provider'
+import Text from 'src/components/Text/Text'
+import { felaRenderer } from 'src/lib'
+import { teams } from 'src/themes'
+
+test('css fallback value', () => {
+  const snapshot = createSnapshot(
+    <EmptyThemeProvider>
+      <Box styles={{ color: 'red' }} />
```

Comment:
This is not testing css fallback value.
- please rename this to something like 'basic styles'.
- add css fallback value test with `styles={{ color: ['red', 'blue'] }}`